docs/source/en/tasks/document_question_answering.mdx

```diff
@@ -0,0 +1,485 @@
+* `query`: the question string - natural language asked question, in several languages
+* `answers`: a list of correct answers provided by annotators
+* `words` and `bounding_boxes`: the results of OCR, which we will not use here
+* `answer`: an answer matched by a different model which we will not use here
+
+Let's leave only English questions, and drop the answer feature which appears to contain predictions by another model.
+
+```py
+>>> updated_dataset = dataset.map(lambda example: {"question": example["query"]["en"]}, remove_columns=["query"])
+>>> updated_dataset = updated_dataset.map(
+...     lambda example: {"answer": example["answers"][0]}, remove_columns=["answer", "answers"]
+... )
+```
+
+Note that the LayoutLMv2 checkpoint that we use in this guide has been trained with `max_position_embeddings = 512` (you can
+find this information in the [checkpoint's `config.json` file](https://huggingface.co/microsoft/layoutlmv2-base-uncased/blob/main/config.json)).
+We can truncate the examples but to avoid the situation where the answer might be at the end of a large document and end up truncated,
+let's remove the few examples where the embedding is likely to end up longer than 512. If most of the documents in your dataset are long, you
+may need to choose a different pretrained model checkpoint.
+
+```py
+>>> updated_dataset = updated_dataset.filter(lambda x: len(x["words"]) + len(x["question"].split()) < 512)
+```
+
+At this point let's also remove the OCR features from this dataset. These are a result of OCR for fine-tuning a different
+model. They would still require some processing if we wanted to use them, as they do not match the input requirements
+of the model we use in this guide. Instead, we can use the [`~LayoutLMv2Processor`] on the original data for both OCR and
+tokenization. This way we'll get the inputs that match model's expected input. If you want to process images manually,
+check out the [`LayoutLMv2` model documentation](../model_doc/layoutlmv2) to learn what input format the model expects.
+
+```py
+>>> updated_dataset = updated_dataset.remove_columns("words")
+>>> updated_dataset = updated_dataset.remove_columns("bounding_boxes")
+```
+
+Finally, the data exploration won't be complete if we don't peek at an image example.
+
+```py
+>>> updated_dataset["train"][11]["image"]
+```
+
+<div class="flex justify-center">
+     <img src="https://imgur.com/VfOeFjc" alt="DocVQA Image Example"/>
+ </div>
+
+## Preprocess the data
+
+The Document Question Answering task is a multimodal task, and you need to make sure that the inputs from each modality
+are preprocessed according to model's expectations. Let's start by loading the [`~LayoutLMv2Processor`], which internally
+combines an image processor that can handle image data and a tokenizer that can encode text data.
+
+```py
+>>> from transformers import LayoutLMv2Processor
+
+>>> processor = LayoutLMv2Processor.from_pretrained(model_checkpoint)
```

Comment:
It would be better to use AutoProcessor here, so this task guide can work with another architecture (as advertised at the top)

Reply:
Fixed in 60cd46474eb9051f3b6c1f1c74e912a0e9ab0f8d